packages/react/passport/src/PassportContext.tsx

```diff
@@ -0,0 +1,187 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import React, {
+  createContext, useCallback, useContext, useEffect, useMemo,
+  useState,
+} from 'react';
+import { setPassportClientId, track } from '@imtbl/metrics';
+import {
+  Passport, PassportModuleConfiguration,
+  UserProfile,
+} from '@imtbl/passport';
+
+type ZkEvmReactContext = {
+  passportInstance: Passport;
+  isLoggedIn: boolean;
+  isLoading: boolean;
+  withWallet: boolean;
+  error: Error | null;
+  getIdToken: Passport['getIdToken'];
+  getAccessToken: Passport['getAccessToken'];
+  getAccounts: () => Promise<string[]>;
```

Comment:
Can I suggest that we expose the connectEvm method instead of `getAccounts` like this? This interface would make it difficult to support something like app-chains in the future